suite/tests/client-interface/drx-scattergather-aarch64.cpp

```diff
@@ -30,6 +30,7 @@
  * DAMAGE.
  */
 
+#include <algorithm>
```

Comment:
I feel it's not easy to review the test values very closely. I think one way to easily catch test value bugs would be to run the test natively also (in addition to under DR and drmemtrace). Native runs would reveal correctness issues with assertion values. Could we add a torunonly_native test?

We do native run tests also when we want to verify our assumptions of kernel behavior.

Reply:
Done. I have been testing the test app by running it natively as I go using 128-bit, 256-bit, and 512-bit vector lengths (all the sizes supported by DynamoRIO) so I know the reference data is correct, but its still good to add a `torunonly_native(...)` test to prove it stays correct.